README.md

```diff
@@ -1 +1,46 @@
-# Future Debian/Ubuntu package for The Lounge
+# Debian/Ubuntu package for The Lounge
+This repository holds out the build scripts that generates our .deb precompiled packages and also tracks Debian-specific issues in relation to the packaging.
+
+## Installation
+If you are looking to simply install The Lounge, please use our pre-compiled binary .deb files available in the releases section of the main project. This section assumes you want to build a Debian package from sources.
+
+```
+# Clone the repository
+$ git clone https://github.com/thelounge/arch-lounge.git
+$ cd deb-lounge
+
+# Call the build script
+$ ./build-package
+```
+
+After this, you should have a nice `.deb` file in the `deb/` output folder! This file can then be installed:
+
+```
+# dpkg -i deb/*.deb
+```
+
+### Configuration
+The default system-wide documentation file is located at `/etc/thelounge/config.js`. Please note that user profiles and their IRC passwords are also stored there (moving to `/var/lib/thelounge` is planned but not yet done), so the directory is only readable by the `thelounge` user.
```

Comment:
The package on npm is `thelounge` but the binary it creates is `lounge`. I would go for consistency with `/etc/thelounge/...`, system user `lounge`, etc.

Also, I wouldn't mention "(moving to `/var/lib/thelounge` is planned but not yet done)" because this hasn't been really discussed on the main project nor it is in the works. I'm not saying I'm against at all, just that it's too early and noise here.